src/setup.rs

```diff
@@ -9,6 +9,9 @@ use std::os::unix::process::ExitStatusExt;
 use std::path::Path;
 use std::process::Command;
 
+const SYSTEMD_MAKEFS_COMMAND: Option<&str> = std::option_env!("SYSTEMD_MAKEFS_COMMAND");
```

Comment:
Something like
```rust
const SYSTEMD_MAKEFS_COMMAND: &str = concat!(env!("SYSTEMD_UTIL_DIR"), "/systemd-makefs");
// or even
const SYSTEMD_MAKEFS_COMMAND: &str = concat!(env!("SYSTEMD_UTIL_DIR", "Define $SYSTEMD_UTIL_DIR to the result of $(pkg-config --variable=systemdutildir systemd) (e.g. /usr/lib/systemd/)"), "/systemd-makefs");
```
would be better, the duplication of logic between here and the makefile could easily go out of sync.